Python/Product/PythonTools/PythonTools/Debugger/ProvideDebugAdapterAttrributes.cs

```diff
@@ -136,6 +148,13 @@ class ProvideDebugAdapterAttribute : RegistrationAttribute {
             adapterKey.SetValue("Assembly", assembly);
             adapterKey.SetValue("Class", className);
             adapterKey.SetValue("CodeBase", $@"$PackageFolder$\{assembly}.dll");
+
+            var customProtocolKey = context.CreateKey($"CLSID\\{_customProtocolExtensionCLSID}");
+            var assembly2 = _customProtocolType.Assembly.GetName().Name;
```

Comment:
I'd rename them to adapterAssembly and customProtocolAssembly, same for className